### PR TITLE
show alert.Message as monitor message in mkr alerts list

### PR DESCRIPTION
**Pull request by astj**
example:
```
aaa 2018-10-26 17:05:52 CRITICAL HTTP CRITICAL: Get https://127.0.0.1/: dial tcp 127.0.0.1:443: connect: connection refused xxxproxy22.local standby [XXX:proxy]
bbb 2018-10-25 16:35:04 CRITICAL connectivity yyybackend10.local working [YYY:backend]
```

Handling:
- display up to 100 runes
- trim at first newline (`[\r\n]`)

### CODE REVIEW

File: alerts.go [hunk 1]
```diff
@@ -208,6 +214,22 @@ func formatExpressionOneline(expr string) string {
...
 }
 
+func formatCheckMessage(msg string) string {
+	truncated := false
+	if index := strings.IndexAny(msg, "\n\r"); index != -1 {
+		msg = msg[0:index]
+		truncated = true
+	}
+	if msgU := utf8string.NewString(msg); msgU.RuneCount() > 100 {
+		msg = msgU.Slice(0, 100)
```
Comment (itchyny) [inline alerts.go]: What's the benefit of using utf8string package, rather than casting to `[]rune`?

**Comment by itchyny** [comment]
LGTM

File: alerts.go (hunk 1 above)
Comment (astj) [reply alerts.go]: Just I forgot about `[]rune` 😱 . Fixed to use `[]rune`.

**Comment by astj** [comment]
Thanks!

**Pull request merged**